apps/web/components/booking/BookingDescription.tsx

```diff
@@ -84,7 +85,7 @@ const BookingDescription: FC<Props> = (props) => {
             {t("requires_confirmation")}
           </div>
         )}
-        {!isBookingPage ? (
+        {!isBookingPage && !props.rescheduleUid ? (
```

Comment:
Don't show available locations of event type when rescheduling (User can't choose between them, so no need to show them)